File: demouilayout-101/src/main/java/algokelvin/demo/uilayout/MainActivity.java
package algokelvin.demo.uilayout;

import android.graphics.Typeface;
import android.os.Bundle;
import android.view.Gravity;

import androidx.appcompat.app.AppCompatActivity;

import algokelvin.demo.uilayout.databinding.ActivityMainBinding;
import ui.algokelvin.uiview.textview.TextViewBorder;
import ui.algokelvin.uiview.textview.TextViewStylish;

public class MainActivity extends AppCompatActivity {
    private final TextViewStylish textViewStylish = new TextViewStylish(this);

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityMainBinding binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        // Text Stylish - Size, Color, Alignment, and Style
        textViewStylish.setTextStylish(
                binding.txtSample, // id binding TextView
                20f, // size Text
                R.color.purple_500, // color Text
                Gravity.CENTER, // alignment Text
                Typeface.DEFAULT_BOLD // Style Text
        );

        // Text Color Mix
        String[] colorArray = {"#800000", "#000080"};
        String[] txtArray = {"Algo", "Kelvin"};
        textViewStylish.setColorCustom(binding.txtSample02, txtArray, colorArray);

        TextViewBorder txtBorder = new TextViewBorder(this,
                R.color.white, // Background color
                R.color.black, // Color Stroke
                1, // Size Stroke
                getSizeDp(getResources(), 8) // Size Radius
        );

    }
}

File: app-implement/src/main/java/com/algokelvin/android/app_implement/MainActivity.kt
package com.algokelvin.android.app_implement

import android.graphics.Typeface
import android.os.Bundle
import android.view.Gravity
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import ui.algokelvin.uiview.textview.TextViewMethod.setColorCustom
import ui.algokelvin.uiview.textview.TextViewMethod.setTextStylish

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        txt_sample.setTextStylish(this, 20f, R.color.purple_500, Gravity.CENTER, Typeface.DEFAULT_BOLD)
        txt_sample02.setTextStylish(this, null, R.color.purple_500, Gravity.CENTER, Typeface.DEFAULT_BOLD)
        txt_sample03.setTextStylish(this, 18f, R.color.purple_500, null, null)

        var colorArray = arrayOf("#800000", "#000080")
        var txtArray = arrayOf("Algo", "Kelvin")
        txt_sample04.setColorCustom(txtArray, colorArray)
        colorArray = arrayOf("#111112", "#FFCDE3")
        txtArray = arrayOf("273", "373")
        txt_sample05.setColorCustom(txtArray, colorArray)
    }
}